src/main/java/seedu/address/model/status/Status.java

```diff
@@ -5,4 +5,27 @@
  */
 public enum Status {
     ACTIVE, INACTIVE, PLAN_TO_APPLY, APPLICATION_DONE, INTERVIEW_SCHEDULED, INTERVIEW_DONE, ACCEPTED, REJECTED;
+
+    private static final String[] validStatuses = new String[] {
+            "active", "inactive", "plan to apply", "plan_to_apply", "application done", "application_done",
+            "interview scheduled", "interview_scheduled", "interview done", "interview_done", "accepted",
+            "rejected"
+    };
+    public static final String MESSAGE_CONSTRAINTS =
+            "Status should only be one of the following strings: "
+            + "active, inactive, plan to apply, plan_to_apply, application done, application_done"
+            + "interview scheduled, interview_scheduled, interview done, interview_done, accepted"
+            + "rejected";
```

Comment:
Was thinking we should review our status enum. Since APPLICATION_DONE all the way to REJECTED would imply the application is ACTIVE, do we need the status ACTIVE?